environment/environment.go

```diff
@@ -30,6 +30,14 @@ func GetDefaultPaastaKubernetesEnvironment() []corev1.EnvVar {
 				},
 			},
 		},
+		{
+			Name: "PAASTA_CLUSTER",
+			ValueFrom: &corev1.EnvVarSource{
+				FieldRef: &corev1.ObjectFieldSelector{
+					FieldPath: "metadata.labels['yelp.com/paasta_cluster']",
```

Comment:
I'm assuming that the `PAASTA_CLUSTER` env var will be empty in the case of missing `'yelp.com/paasta_cluster'` label, right?

Reply:
yep